engine/src/test/java/org/camunda/bpm/engine/test/api/mgmt/telemetry/TelemetryDynamicDataTest.java

```diff
@@ -141,31 +144,38 @@ public void shouldCountAfterCleaning() {
   }
 
   @Test
-  public void shouldCollectCommandsDataOnlyWhenTelemetryEnabled() {
+  public void shouldResetCollectedCommandsDataWhenTelemetryEnabled() {
     // given default telemetry data and empty telemetry registry
 
-    // execute commands
+    // when execute commands
     managementService.getHistoryLevel();
     managementService.getLicenseKey();
+
+    // then
     Map<String, CommandCounter> commands = configuration.getTelemetryRegistry().getCommands();
-    assertThat(commands.size()).isZero();
+    assertThat(commands.size()).isEqualTo(3);
+    assertThat(commands.get("GetLicenseKeyCmd").get()).isEqualTo(1);
+    assertThat(commands.get("GetHistoryLevelCmd").get()).isEqualTo(1);
+    assertThat(commands.keySet()).contains("DeleteMetricsCmd");
 
     // when
     managementService.toggleTelemetry(true);
     // execute commands
-    managementService.getHistoryLevel();
-    managementService.getLicenseKey();
+    managementService.getTelemetryData();
+    managementService.getTelemetryData();
+    managementService.isTelemetryEnabled();
 
-    // when
+    // then
     assertThat(commands.size()).isEqualTo(3);
-    String [] expectedExcutedCommands = {"GetLicenseKeyCmd","GetHistoryLevelCmd","TelemetryConfigureCmd"};
-    assertThat(commands.keySet()).isSubsetOf(expectedExcutedCommands);
+    assertThat(commands.get("GetTelemetryDataCmd").get()).isEqualTo(2);
+    assertThat(commands.get("IsTelemetryEnabledCmd").get()).isEqualTo(1);
+    assertThat(commands.get("TelemetryConfigureCmd").get()).isEqualTo(1);
   }
 
 
   @Test
   @Deployment(resources = { "org/camunda/bpm/engine/test/api/oneTaskProcess.bpmn20.xml" })
-  public void shouldNotCollectMetricsDataWhenTelemetryDisabled() {
+  public void shouldCollectMetricsDataWhenTelemetryDisabled() {
```

Comment:
The test is a strong candidate for removal: It does not test API and the impact on API functionality is covered by other tests (e.g. `ManagementServiceGetTelemetryDataTest.shouldCollectMetrics_TelemetryEnabled`). What do you think?

Reply:
I agree. I will remove it.